nav-challenge/src/components/layout/Layout.scss

```diff
@@ -1,8 +1,10 @@
+$background-color: rgb(216, 255, 110);
```

Comment:
Can we rename this to the specific color instead of it's purpose?
e.g. `$lime-color = rgb(216, 255, 110);`
This is so that we can avoid a collision if we have multiple `background-color`s for different elements

Reply:
Also it is a good idea to define a `variables.scss` file to store all the variables in one place